subworkflows/local/call_repeat_expansions.nf

```diff
@@ -0,0 +1,25 @@
+//
+// Run ExpansionHunter
+//
+
+params.expansionhunter_options = [:]
+
+include { EXPANSIONHUNTER } from '../../modules/nf-core/modules/expansionhunter/main'  addParams( options: params.expansionhunter_options )
+
+workflow CALL_REPEAT_EXPANSIONS {
+    take:
+        bam 				// channel: [ val(meta), path(bam), path(bai) ]
+        fasta 			// channel: /path/to/genome.fasta
+        variant_catalog		// channel: /path/to/variant_catalog.json
+
+    main:
+
+        ch_versions = Channel.empty()
+        
```

Comment:
```suggestion

```

Reply:
this should remove the linting errors :D but you could also test it locally before pushing by using this: `nf-core lint`